shell/packages/sandstorm-db/db.js

```diff
@@ -639,6 +639,30 @@ SetupSession = new Mongo.Collection("setupSession");
 //   creationDate: Date object indicating when this session was created.
 //   hashedSessionId: the sha256 of the secret session id that was returned to the client
 
+const DesktopNotifications = new Mongo.Collection("desktopNotifications");
+// Responsible for very short-lived queueing of desktop notification information.
+// Entries are removed when they are ~30 seconds old.  This collection is a bit
+// odd in that it is intended primarily for edge-triggered communications, but
+// Meteor's collections aren't really designed to support that organization.
+// Fields for each :
+//
+//   _id: String.  Used as the tag to coordinate notification merging between browser tabs.
+//   creationDate: Date object. indicating when this notification was posted.
+//   accountId: String. Account id to which this notification was published.
```

Comment:
All of our other collections that describe items owned by specific accounts name the field `userId` rather than `accountId`. I know that `accountId` is less ambiguous, but the inconsistency can lead to mistakes when writing Mongo queries where I can't remember which field name to use. Maybe we should stick with `userId`?